main/lsp/test/lsp_preprocessor_test.cc

```diff
@@ -34,6 +34,7 @@ options::Options makeOptions(string_view rootPath) {
     opts.rawInputDirNames.emplace_back(string(rootPath));
     opts.runLSP = true;
     opts.fs = make_shared<sorbet::test::MockFileSystem>(rootPath);
+    opts.allowedExtensions = {".rb", ".rbi"};
```

Comment:
Let's set this as a default in options.h, and delete the logic from options.cc that sets this if no options is passed. That way we don't have to worry about places like this that create a bare options struct.